### PR TITLE
EOS-24442: Utils Code optimize

### PR DESCRIPTION
**Pull request by suryakumar1024**
# Problem Statement
- Updted cortx.conf file hardcoded path with proper path
-  Added return statement to get_utils_path

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -66,8 +66,12 @@ def _get_utils_path() -> str:
         """ Gets install path from cortx.conf and returns utils path """
         install_path = CortxConf.get(key='install_path')
         if not install_path:
+            local_storage_path = CortxConf.get_storage_path('local')
+            config_url = "%s://%s" % ('json', \
```
Comment (rohit-k-dwivedi) [inline py-utils/src/setup/utils.py:70]: minor: use single inverted comma in "%s://%s"

**Review by selvakumaar5496 — APPROVED** [review]

File: py-utils/test/discovery/test_discovery.py [hunk 2]
```diff
@@ -24,10 +24,12 @@
 from cortx.utils.discovery import Discovery
```
Comment (rohit-k-dwivedi) [inline py-utils/test/discovery/test_discovery.py:24]: from cortx.utils import const can be removed

**Review by vastradparayya — APPROVED** [review]

**Review by rohit-k-dwivedi — APPROVED** [review]

File: py-utils/test/discovery/test_discovery.py (hunk 2 above)
Comment (suryakumar1024) [reply py-utils/test/discovery/test_discovery.py:23]: Updated in recent patch

**Pull request merged**